subprojects/docs/src/snippets/valueProviders/externalProcessProvider/groovy/build.gradle

```diff
@@ -0,0 +1,3 @@
+def gitVersion = providers.exec {
+    commandLine("git", "--version")
```

Comment:
:lipstick: Maybe `git describe` would be a more actionable example? No strong opinion

Reply:
I tried that but the test harness for samples has no git repo so the call just fails, failing the test.

Reply:
I didn't think about that. The current samples are fine :+1: